sycl/source/detail/error_handling/enqueue.cpp

```diff
@@ -0,0 +1,179 @@
+//===-- enqueue.cpp - SYCL error handling of enqueue operations -*- C++ -*-===//
```

Comment:
Suggest renaming the file to something like enqueue_kernel

Reply:
Done